src/lumigo_tracer/spans_container.py

```diff
@@ -73,6 +74,7 @@ def __init__(
         self.transaction_id = transaction_id
         self.max_finish_time = max_finish_time
         self.base_msg = {
+            "lambda_container_id": SpansContainer.lambda_container_id,
```

Comment:
do you want it in the HTTP spans as well?
If no, you should put it under seelf.function_span

Reply:
Yes for all connections